frontend/src/pages/Tomorrow.jsx

```diff
@@ -62,26 +58,24 @@ function Tomorrow() {
           <img src={Pic3} alt="salade poke" />
         </div>
         <div className="paragraph">
-          <h2>Opter pour une Alimentation Végétarienne et Végétalienne</h2>
+          <h2>Opting for a Vegetarian and Vegan Diet</h2>
           <p>
-            La production de viande a un impact environnemental significatif en
-            termes d'utilisation de ressources et d'émissions de gaz à effet de
-            serre. En incorporant davantage de repas à base de plantes dans
-            votre alimentation, vous réduisez votre empreinte carbone et
-            contribuez à la préservation des écosystèmes.
+            Meat production has a significant environmental impact in terms of
+            resource use and gas emissions tight. By incorporating more
+            plant-based meals into your diet, you reduce your carbon footprint
+            and contribute to the preservation of ecosystems.
           </p>
         </div>
       </div>
 
       <div className="text-container">
         <div className="paragraph">
-          <h2>Privilégier les Aliments Biologiques</h2>
+          <h2>Favor Organic Foods</h2>
           <p>
-            Les aliments biologiques sont produits sans l'utilisation de
-            pesticides chimiques et d'engrais synthétiques, ce qui les rend plus
-            respectueux de l'environnement. En choisissant des produits
-            biologiques, vous soutenez une agriculture durable et préservez la
-            biodiversité.
+            Organic foods are produced without the use of chemical pesticides
+            and synthetic fertilizers, making them more eco-friendly. When
+            choosing products organic, you support sustainable agriculture and
+            preserve the biodiversity.
```

Comment:
Cadeau => Organic foods are produced without the use of chemical pesticides and synthetic fertilizers, making them more environmentally friendly. By choosing organic products, you support sustainable agriculture and preserve biodiversity.